src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -731,9 +741,27 @@ private List<InetAddress> lookupPublic(String host) throws UnknownHostException
                 .writeTimeout(20, TimeUnit.SECONDS)
                 .hostnameVerifier(noopHostnameVerifier.isPresent() && noopHostnameVerifier.get()
                         ? (hostname, sslSession) -> true
-                        : OkHostnameVerifier.INSTANCE
-                )
-                .build();
+                        : OkHostnameVerifier.INSTANCE);
+
+        if (proxyHost != "https://example.com") {
```

Comment:
```suggestion
        if (!proxyHost.isEmpty() && !proxyHost.equals("https://example.com")) {
```
You need to use the `equals()` method attached to a string. It would probably be a good idea to check if it's empty as well, just in case someone sets it to `""` in an attempt to disable the feature.